rolemanager/core/config.py

```diff
@@ -18,14 +18,13 @@
     },
     "reactroles": {
         "message_cache": {},
-        "channels": [],
         "enable": True,
     },
 }
 
 
 # TODO: Deprecate
-def _resolve_migration(data: Dict[str, Any]) -> bool:
+def _resolve_migration(data) -> bool:
```

Comment:
```suggestion
def _resolve_migration(data: Dict[str, Any]) -> bool:
```

Restore this back. Removed by mistake.